imap_processing/tests/lo/test_lo_l1b.py

```diff
@@ -503,3 +512,67 @@ def test_set_bad_times():
 
     # Assert
     np.testing.assert_array_equal(l1b_de["badtimes"], expected_bad_times)
+
+
+@pytest.mark.external_kernel
+@pytest.mark.use_test_metakernel("imap_ena_sim_metakernel.template")
+def test_set_direction():
+    # Arrange
+    l1b_de = xr.Dataset(
+        {},
+        coords={
+            "epoch": [
+                7.9794907049e17,
+                7.9794907153e17,
+                7.9794907254e17,
+            ],
+        },
+    )
+    # Need to better verify these values. Latitudes are likely to
+    # be 0, but not sure what values to expect from Longitudes.
+    # The expected values are based on what I'm getting from the
+    # SPICE tools.
```

Comment:
We are spinning with a period of 15 seconds, or 24 degrees/s. So you could probably choose an epoch 1 second later and see if it is 24 degrees off? It looks like you are using 1.04 seconds currently.

Reply:
That's a great idea, I didn't think to use the spin period for this. Thanks!